docs/topics/data-analysis/data-analysis-notebooks-output-formats.md

```diff
@@ -1,35 +1,230 @@
+* If your code contains errors, Kotlin Notebook displays an error message and a traceback, providing insights for debugging.
+
+### Rich text
+
+Use Markdown types of cells to get rich text output. It supports various types of formatting with Markdown and HTML markup.
+HTML can even contain CSS and JavaScript. This way, your cell's output can include lists, tables, font styles, code blocks,
+and more:
+
+```none
+## Line magics
+ 
+| Spell                              | Description                                                                                                      | Example                                                                               | 
+|------------------------------------|------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------|
+| <code>%use</code>                  | Injects code for supported libraries: artifact resolution, default imports, initialization code, type renderers. | <code>%use klaxon(5.5), lets-plot</code>                                              |                                          
+| <code>%trackClasspath</code>       | Logs any changes of current classpath. Useful for debugging artifact resolution failures.                        | <code>%trackClasspath [on |off]</code>                                                |
+| <code>%trackExecution </code>      | Logs pieces of code that are going to be executed. Useful for debugging of libraries support.                    | <code>%trackExecution [all|generated|off]</code>                                      |           
+| <code>%useLatestDescriptors</code> | Use latest versions of library descriptors available. By default, bundled descriptors are used.                  | <code>%useLatestDescriptors [on|off]</code>                                           |
+| <code>%output</code>               | Output capturing settings.                                                                                       | <code>%output --max-cell-size=1000 --no-stdout --max-time=100 --max-buffer=400</code> |
+| <code>%logLevel</code>             | Set logging level.                                                                                               | <code>%logLevel [off|error|warn|info|debug]</code>                                    |
+
+<ul><li><a href="https://github.com/Kotlin/kotlin-jupyter/blob/master/docs/magics.md">Learn more detailes about line magics</a>.</li>
+<li><a href="https://github.com/Kotlin/kotlin-jupyter/blob/master/docs/magics.md">See the full list of supported libraries</a>.</li></ul>
+```
+
+![Rich text in Markdown cells](markdown-cells-output.png){width=700}
+
+## HTML
+
+Kotlin Notebook can render HTML directly, showing scripts or even embedding websites:
```

Comment:
Please mention that for scripts to work notebook should be marked as Trusted
<img width="348" alt="image" src="https://github.com/JetBrains/kotlin-web-site/assets/20319593/56d6a151-3395-41fa-a806-d568cfa0c72e">